### ISSUE TITLE
Disable icmp redirects inside rancher-metadata container

### ISSUE THREAD

**Issue by leodotcloud** [rancher/rancher#9548]
Packet capture inside rancher-metadata revealed these packets:

```
23:34:10.719838 IP (tos 0xc0, ttl 64, id 5681, offset 0, flags [none], proto ICMP (1), length 268)
    172.17.0.1 > 169.254.169.250: ICMP redirect 10.42.84.44 to host 10.42.84.44, length 248
        IP (tos 0x0, ttl 63, id 61623, offset 0, flags [DF], proto TCP (6), length 240)
    169.254.169.250.80 > 10.42.84.44.39662: Flags [P.], cksum 0xb331 (incorrect -> 0x6179), seq 41324739:41324927, ack 3950051199, win 930, options [nop,nop,TS val 8677510 ecr 8672510], length 188: HTTP, length: 188
        HTTP/1.1 200 OK
        Access-Control-Allow-Origin: *
        Date: Tue, 01 Aug 2017 23:34:10 GMT
        Content-Length: 39
        Content-Type: text/plain; charset=utf-8

        "1873-4f3c5c96fb170da7fa8781d4ac55192c"[!http]
        0x0000:  02fb 35f7 b8ae 0242 7763 8e6d 0800 45c0  ..5....Bwc.m..E.
        0x0010:  010c 1631 0000 4001 62f5 ac11 0001 a9fe  ...1..@.b.......
        0x0020:  a9fa 0501 fe21 0a2a 542c 4500 00f0 f0b7  .....!.*T,E.....
        0x0030:  4000 3f06 9801 a9fe a9fa 0a2a 542c 0050  @.?........*T,.P
        0x0040:  9aee 0276 90c3 eb70 ff7f 8018 03a2 b331  ...v...p.......1
        0x0050:  0000 0101 080a 0084 6886 0084 54fe 4854  ........h...T.HT
        0x0060:  5450 2f31 2e31 2032 3030 204f 4b0d 0a41  TP/1.1.200.OK..A
        0x0070:  6363 6573 732d 436f 6e74 726f 6c2d 416c  ccess-Control-Al
        0x0080:  6c6f 772d 4f72 6967 696e 3a20 2a0d 0a44  low-Origin:.*..D
        0x0090:  6174 653a 2054 7565 2c20 3031 2041 7567  ate:.Tue,.01.Aug
        0x00a0:  2032 3031 3720 3233 3a33 343a 3130 2047  .2017.23:34:10.G
        0x00b0:  4d54 0d0a 436f 6e74 656e 742d 4c65 6e67  MT..Content-Leng
        0x00c0:  7468 3a20 3339 0d0a 436f 6e74 656e 742d  th:.39..Content-
        0x00d0:  5479 7065 3a20 7465 7874 2f70 6c61 696e  Type:.text/plain
        0x00e0:  3b20 6368 6172 7365 743d 7574 662d 380d  ;.charset=utf-8.
        0x00f0:  0a0d 0a22 3138 3733 2d34 6633 6335 6339  ..."1873-4f3c5c9
        0x0100:  3666 6231 3730 6461 3766 6138 3738 3164  6fb170da7fa8781d
        0x0110:  3461 6335 3531 3932 6322                 4ac55192c"
```

**Comment by leodotcloud** [comment]
Similar to https://github.com/rancher/rancher/issues/8203

**Comment by leodotcloud** [comment]
Has been merged into branch: `rancher:network-services-v025`. 
Available in network-services v0.2.5

**Comment by sangeethah** [comment]
Tested with rancher-server - v1.6.7-rc1 which comes network-services v0.2.5 having rancher/network-manager:v0.7.7 and rancher/metadata:v0.9.3.

Automated validation runs passed and did not report any few failure.
